modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/scim2/rest/api/SCIM2GroupTest.java

```diff
@@ -188,4 +209,163 @@ public void getErrorGroupIdPermissions() throws IOException {
                 .assertThat()
                 .statusCode(HttpStatus.SC_NOT_FOUND);
     }
+
+    @Test(dependsOnMethods = "testGETGroupDetails")
+    public void testGetGroupsAfterRemovingHybridRoleOfAMember() throws Exception {
+
+        final String APPLICATION_NAME = "my-sp";
+        final String APPLICATION_ROLE_NAME = "Application/" + APPLICATION_NAME;
+        final String USERNAME_OF_THE_NEW_USER = USER_STORE_DOMAIN
+                + "/sampleSecondaryUser";
```

Comment:
Any specific reason to have these as local constants rather than defining globally?

Reply:
Added as class level constants in https://github.com/wso2/product-is/pull/7439/commits/5f62c85aab90f9520469d3a7da2fb302dbdfcc27